src/main/java/uk/gov/hmcts/darts/transcriptions/exception/TranscriptionApiError.java

```diff
@@ -22,6 +22,16 @@ public enum TranscriptionApiError implements DartsApiError {
         "102",
         HttpStatus.BAD_REQUEST,
         "Unexpected transcription status for this workflow"
+    ),
+    BAD_REQUEST_TRANSCRIPTION_TYPE(
+        "103",
+        HttpStatus.BAD_REQUEST,
+        "Unexpected transcription type for this workflow"
+    ),
+    TRANSCRIPTION_WORKFLOW_ACTION_INVALID(
```

Comment:
Please decide what should happen with the error I added for my basic validation - whether it will be replaced by 409 - then please update the OAS error responses in your next PR

Error: response status is 400
Response body
```
{
  "type": "TRANSCRIPTION_102",
  "title": "Unexpected transcription status for this workflow",
  "status": 400
}
```

Reply:
ok